lib/src/includes/signed_video_sign.h

```diff
@@ -118,6 +118,37 @@ signed_video_add_nalu_for_signing(signed_video_t *self,
     const uint8_t *nalu_data,
     size_t nalu_data_size);
 
+/**
+ * @brief Updates Signed Video, with a H26x NALU, for signing with timestamp
+ *
+ * For information regarding the function please reference the signed_video_add_nalu_for_signing.
+ * This function is an extention of that function but with the addition of a timestamp.
+ *
+ * The timestamp parameter is user defined, meaning that it is up to the implementer of the library
```

Comment:
I can see some drawbacks using a truly user defined timestamp. For example, if an end user gets hold of a file of a signed video recording. How do that person know how to interpret the timestamp?
Are there any drawbacks using a predefined timestamp format?

Reply:
Modified the description to clearly state that the timestamp should be a Unix timestamp in UTC format.